Description: Some nasty typos fixed to silence lintian et al.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

Index: Mesa/src/mesa/main/debug.c
===================================================================
--- Mesa.orig/src/mesa/main/debug.c
+++ Mesa/src/mesa/main/debug.c
@@ -43,7 +43,7 @@ const char *_mesa_prim_name[GL_POLYGON+4
    "GL_QUAD_STRIP",
    "GL_POLYGON",
    "outside begin/end",
-   "inside unkown primitive",
+   "inside unknown primitive",
    "unknown state"
 };
 
Index: Mesa/src/mesa/shader/arbprogparse.c
===================================================================
--- Mesa.orig/src/mesa/shader/arbprogparse.c
+++ Mesa/src/mesa/shader/arbprogparse.c
@@ -127,8 +127,8 @@
  *
  *    Cosmetic Stuff
  *    -----------------------------------------------------
- * 	- remove any leftover unused grammer.c stuff (dict_ ?)
- * 	- fix grammer.c error handling so its not static
+ * 	- remove any leftover unused grammar.c stuff (dict_ ?)
+ * 	- fix grammar.c error handling so its not static
  * 	- #ifdef around stuff pertaining to extentions
  *
  *    Outstanding Questions:
@@ -1673,7 +1673,7 @@
    if (found) {
       error_msg = (char *)
          _mesa_malloc (_mesa_strlen ((char *) attrib_var->name) + 40);
-      _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+      _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                      attrib_var->name);
 
       _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -1864,7 +1864,7 @@
    if (found) {
       char *error_msg = (char *)
          _mesa_malloc (_mesa_strlen ((char *) param_var->name) + 40);
-      _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+      _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                      param_var->name);
 
       _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -1972,7 +1972,7 @@
       if (found) {
          char *error_msg = (char *)
             _mesa_malloc (_mesa_strlen ((char *) temp_var->name) + 40);
-         _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+         _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                         temp_var->name);
 
          _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -2023,7 +2023,7 @@
    if (found) {
       char *error_msg = (char *)
          _mesa_malloc (_mesa_strlen ((char *) output_var->name) + 40);
-      _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+      _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                      output_var->name);
 
       _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -2057,7 +2057,7 @@
    if (found) {
       char *error_msg = (char *)
          _mesa_malloc (_mesa_strlen ((char *) temp_var->name) + 40);
-      _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+      _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                      temp_var->name);
 
       _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -2106,7 +2106,7 @@
       if (found) {
          char *error_msg = (char *)
             _mesa_malloc (_mesa_strlen ((char *) temp_var->name) + 40);
-         _mesa_sprintf (error_msg, "Duplicate Varible Declaration: %s",
+         _mesa_sprintf (error_msg, "Duplicate Variable Declaration: %s",
                         temp_var->name);
 
          _mesa_set_program_error (ctx, Program->Position, error_msg);
@@ -3904,7 +3904,7 @@
       grammar_get_last_error ((GLubyte *) error_msg, 300, &error_pos);
       _mesa_set_program_error (ctx, error_pos, error_msg);
       _mesa_error (ctx, GL_INVALID_OPERATION,
-                   "glProgramString(Error loading grammer rule set)");
+                   "glProgramString(Error loading grammar rule set)");
       return GL_FALSE;
    }
 
